helpers/api_helper/json_api_response.go

```diff
@@ -0,0 +1,32 @@
+package api_helper
+
+import "github.com/gin-gonic/gin"
+
+type ErrorResponseObject struct {
+	Title  string `json:"title"`
+	Detail string `json:"detail"`
+	Status int    `json:"status"`
+}
+
+func (e *ErrorResponseObject) ConstructErrorResponse() gin.H {
```

Comment:
`Construct` is not very common naming, how about to use a more generic term like `BuildErrorResponse` (there is also such a thing called [the Builder pattern](https://golangbyexample.com/builder-pattern-golang/)). But maybe here since it's returning a JSON based on the attributes params, how about just:

```suggestion
func (e *ErrorResponseObject) NewErrorResponse() gin.H {
```

Then in the controller, it twould be:

```go
c.JSON(errorResponse.Status, errorResponse.NewErrorResponse())
```

Reply:
Make sense. Fixed in 23688b8db37ebc32331c1a92c71308cdd78010d9